cinema/serializers.py

```diff
@@ -1 +1,114 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    MovieSession,
+    Order,
+    Ticket,
+    CinemaHall,
+    Movie
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row", "capacity")
```

Comment:
The `capacity` field is included in the `CinemaHallSerializer`, but it is not clear if this field exists in the `CinemaHall` model. Ensure that the `CinemaHall` model has a `capacity` field, or remove it from the serializer if it does not exist.